plugins/workspace-backpack/src/backpack.ts

```diff
@@ -553,7 +556,39 @@ export class Backpack
    * @param block The block to be removed from the backpack.
    */
   removeBlock(block: Blockly.Block) {
-    this.removeItem(this.blockToJsonString(block));
+    if (isBackpackable(block)) this.removeBackpackable(block);
+  }
+
+  /**
+   * Returns whether the backpack contains a duplicate of the provided
```

Comment:
please add the jsdoc. you can just add the parameter, i'm pretty sure we don't require it to have a description, but it being in jsdoc is important (this will matter for generated reference docs which will happen eventually)

Reply:
Ah, we have it configured exactly the opposite way in core.

Reply:
oh weird, we should update it in core. It needs to be in the jsdoc to show up in the generated reference doc I'm pretty sure. will file an issue for that

Reply:
I think I've already filed an issue about it. So you might want to check.